class-14.md

```diff
@@ -0,0 +1,85 @@
+# Intro to Blockchain Development
+
+### Class 14 - Wallets 
+
+We'll jump back to how cryptocurrency wallets are implemented, to give a better idea for users.
+
+Reading & Videos:
+
+- [Mastering Bitcoin Chapter 5](https://github.com/bitcoinbook/bitcoinbook/blob/develop/ch05.asciidoc)* - A deeper look at how a wallet works in bitcoin
+
+
+
+### Intro To Wallets
+
+We looked before briefly at how funds are stored in a wallet, different community proposals to enhance storage of funds, and how SPV worked to rapidly verify transactions. Now, we will dig a bit farther into the process, and explore how to interact with transactions at a programmatic level as well.
+
+To start with, we'll be using [bitcoinjs-lib](https://github.com/bitcoinjs/bitcoinjs-lib). You can install it using:
+
+```bash
+npm install bitcoinjs-lib
+```
+
+There are some packages in other languages, but this one is built out pretty well and easy to follow. There are some python libraries as well ([bit](https://github.com/ofek/bit), [python-bitcoinlib](https://github.com/petertodd/python-bitcoinlib)).
```

Comment:
~There are some packages in other languages, but this one is built out pretty well and easy to follow. There are some python libraries as well ([bit](https://github.com/ofek/bit), [python-bitcoinlib](https://github.com/petertodd/python-bitcoinlib)).~

There are alternative libraries for other languages such as python, as seen here: ([bit](https://github.com/ofek/bit), [python-bitcoinlib](https://github.com/petertodd/python-bitcoinlib))